### PR TITLE
create form on bootstrap

### PR DESCRIPTION
**Pull request by Yuriy777**

### CODE REVIEW

File: src/app/general-form-bootstrap/form/general-form-bootstrap.component.html [hunk 1]
```diff
@@ -1,3 +1,1203 @@
-<div style="text-align:center">
-  <button class="mr-20" routerLink="/">Material</button>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```
Comment (Betrozov) [inline src/app/general-form-bootstrap/form/general-form-bootstrap.component.html]: @Yuriy777 Why do you need it?

File: src/app/general-form-bootstrap/form/general-form-bootstrap.component.html [hunk 2]
```diff
@@ -1,3 +1,1203 @@
...
+
+  <section class="general-form">
+    <form>
+      <header class="form-header">
+        <div class="container-fluide">
+         <div class="row">
+           <div class="col-md-4">
+             <div class="row">
+               <div class="logo brand col-md-12 col-xl-6 align-self-center">
+                 <img class="img-fluid" src="../../../assets/images/logo.png" alt="truck center">
```
Comment (Betrozov) [inline src/app/general-form-bootstrap/form/general-form-bootstrap.component.html]: @Yuriy777 please use absolute path

File: src/app/general-form-bootstrap/form/general-form-bootstrap.component.html [hunk 3]
```diff
@@ -1,3 +1,1195 @@
...
+               </div>
+             </div>
+           </div>
+           <h1 class="col-md-5 col-xl-6 align-self-center text-center">Used truck appraisal</h1>
+           <div class="col-md-3 col-xl-2 align-self-center">
+            <div class=" row header-select">
+              <label class="col-sm-6" for="exampleFormControlSelect1">Branch: </label>
+              <div class="col-sm-6">
+                <select class="form-control " id="exampleFormControlSelect1">
+                  <option></option>
```
Comment (Betrozov) [inline src/app/general-form-bootstrap/form/general-form-bootstrap.component.html]: @Yuriy777 Why is it empty?

File: src/app/general-form-bootstrap/form/general-form-bootstrap.component.html [hunk 4]
```diff
@@ -1,3 +1,1195 @@
...
+              <div class="form-group">
+                <label class="field-title" for="engineSerial">Engine Serial #</label>
+                <input type="text" class="form-control" id="engineSerial">
+              </div>
+            </div>
+            <div class="col-md-4 col-lg-2">
+              <div class="form-group">
+                <label class="field-title" for="epaEngineRating">EPA Engine Rating</label>
+                <select class="form-control" id="epaEngineRating">
+                  <option></option>
```
Comment (Betrozov) [inline src/app/general-form-bootstrap/form/general-form-bootstrap.component.html]: @Yuriy777 Why is it empty?
Please check it everywhere

File: src/app/general-form-bootstrap/form/general-form-bootstrap.component.html [hunk 5]
```diff
@@ -1,3 +1,1195 @@
...
+                  <label></label>
+                  <div class="form-group row">
+                    <label class="col-md-4 col-xl-2" for="make1">Make:</label>
+                    <div class="col-md-8 col-xl-10">
+                      <input type="text" class="form-control" id="make1">
+                    </div>
+                  </div>
+                </div>
+                <div class="col-sm-6">
+                  <label></label>
```
Comment (Betrozov) [inline src/app/general-form-bootstrap/form/general-form-bootstrap.component.html]: @Yuriy777 Why do you need empty label?

File: src/app/general-form-bootstrap/form/general-form-bootstrap.component.html (hunk 3 above)
Comment (Yuriy777) [reply src/app/general-form-bootstrap/form/general-form-bootstrap.component.html]: I think, fields with select should be empty by default

Comment (Betrozov) [reply src/app/general-form-bootstrap/form/general-form-bootstrap.component.html]: I understood

File: src/app/general-form-bootstrap/form/general-form-bootstrap.component.html [hunk 6]
```diff
@@ -1,3 +1,1191 @@
...
+
+  <section class="general-form">
+    <form>
+      <header class="form-header">
+        <div class="container-fluide">
+         <div class="row">
+           <div class="col-md-4">
+             <div class="row">
+               <div class="logo brand col-md-12 col-xl-6 align-self-center">
+                 <img class="img-fluid" src="./assets/images/logo.png" alt="truck center">
```
Comment (Betrozov) [inline src/app/general-form-bootstrap/form/general-form-bootstrap.component.html]: @Yuriy777 `./assets/images/logo.png` -> `/assets/images/logo.png`

**Pull request merged**